paywall/src/__tests__/data-iframe/Mailbox/sanitizeBlockchainData.test.ts

```diff
@@ -40,60 +39,15 @@ jest.mock('@unlock-protocol/unlock-js', () => {
   }
 })
 
-describe('Mailbox - getDataToSend', () => {
+describe('Mailbox - sanitizeBlockchainData', () => {
```

Comment:
This test was named incorrectly, which made it difficult to find which file the failing test was in